test/jdk/java/nio/channels/Selector/LotsOfInterrupts.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I guess you should credit SAP here.

Reply:
Alan wrote the test. I added him as contributor.